plugins/com.google.cloud.tools.eclipse.appengine.facets/META-INF/MANIFEST.MF

```diff
@@ -15,9 +15,12 @@ Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.5.2",
  org.eclipse.jst.j2ee,
  org.eclipse.jst.j2ee.web,
  org.eclipse.m2e.maven.runtime,
- org.eclipse.swt
+ org.eclipse.ui.navigator,
+ org.eclipse.ui;bundle-version="3.108.1"
```

Comment:
Why do we need this specific version here?

Reply:
Will drop.  I suppose since we say we have a minimum platform of Neon, perhaps we should be putting in neon versions across our bundles.  But that's a big change.

Reply:
Ping